Fix implicit declarations caused by missing headers and
failures caused by -Werror=format-security checks

--- videorbits-2.205/src/pchirp2nocrop.c
+++ videorbits-2.205/src/pchirp2nocrop.c
@@ -1,5 +1,6 @@
 #include <stdlib.h>
 #include <math.h>
+#include <string.h>
 #include "PNMImageOffsetable.h"
 #include "mat_util.h"
 #include "boundingbox.h"
--- videorbits-2.205/src/par.c
+++ videorbits-2.205/src/par.c
@@ -111,7 +111,7 @@
 
   /* - - Write file-type number - - */ 
   sprintf(magic_number, "P%d\n", data_type);
-  fprintf(file_ptr, magic_number); 
+  fprintf(file_ptr, "%s", magic_number); 
 
   /* - - put comments - - */  /* !!!!!! break up long comments */
   fprintf(file_ptr, "# %s\n", comments);  
